src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java

```diff
@@ -95,7 +95,7 @@ public CourseDTO createCourse(final CourseInitialData courseInit) {
       courseInit.getCourseName(),
       courseInit.getSemester(),
       courseInit.getDescription(),
-      false,
+      true,
```

Comment:
As I've written in the clone courses PR: Please use a constant instead.

Reply:
Ok, but then I revert this and this should be done in the clone course PR. We do not need 2 constants for this

Reply:
Ok, then I will do this. This PR should go through after I added this.